packages/fuels-core/src/types/wrappers/coin_type.rs

```diff
@@ -43,4 +43,11 @@ impl CoinType {
             CoinType::Message(_) => BASE_ASSET_ID,
         }
     }
+
+    pub fn owner(&self) -> &Bech32Address {
+        match self {
+            CoinType::Coin(coin) => &coin.owner,
+            CoinType::Message(message) => &message.sender,
```

Comment:
Should be changed to `recipient' since the `sender` is no longer the owner of the message.